tests/test_diff.py

```diff
@@ -269,7 +269,7 @@ def test_diff_reprojection(
     repo_version, output_format, data_working_copy, geopackage, cli_runner
 ):
     """ diff the working copy against HEAD """
-    data_archive = "points2" if repo_version == 2 else "points"
+    data_archive = "points" if repo_version == 2 else "points"
```

Comment:
and here. all the tests in this file?

Reply:
Done - this was a bit lax, I must've accidentally "fixed" this file using a search/replace instead of doing it the proper way.